test/fact_importer_test_helper.rb

```diff
@@ -154,8 +151,6 @@ def good_facts
 
     def ignored_facts
       {
-        'macaddress_vnet4' => 'fe:54:00:59:4e:bf',
-        'mtu_vnet4' => '1500',
```

Comment:
Why are you removing these? These should be continued to be filtered out. If you stop ignoring these, it's a regression.

Reply:
Oh I think this is the culprit of the problem! We actually rely on the fact that filter "vnet*" actually filters "anything_vnet*" so we get rid of these facts too.

My concern is, if we start filtering just beginning of lines, users with older facter will get many more facts and this could cause regressions.